packages/rocketchat-livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -0,0 +1,80 @@
+Template.visitorForward.helpers({
+	visitor() {
+		return Template.instance().visitor.get();
+	},
+	hasDepartments() {
+		return LivechatDepartment.find({ enabled: true }).count() > 0;
+	},
+	departments() {
+		return LivechatDepartment.find({ enabled: true });
+	},
+	agents() {
+		return AgentUsers.find({ _id: { $ne: Meteor.userId() } }, { sort: { name: 1, username: 1 } });
```

Comment:
I thought about that, but if the other agent is `invisible` (or unavailable for livechat) but is on your side? I thought you'd like to transfer to him anyway =)

I thought then add the online agents first.. what you think?